tests/unit/build-order-actions.spec.js

```diff
@@ -164,12 +176,12 @@ test(`lineItems
     },
   )
 
-  const actions = buildOrderActions.lineItems(order)
+  const actions = buildOrderActions.lineItems(order, orderSample())
```

Comment:
@hisabimbola is there any reason why is the `orderSample` a function which returns a static JSON object?